agent/src/CheckSignature.php

```diff
@@ -37,8 +40,11 @@ public function start(): void
 
     private function check(): void
     {
+        clearstatcache(true, $this->signaturePath);
```

Comment:
For poorly designed APIs, e.g., those that accept boolean flags, I tend to like to have named parameters so we know what the boolean is doing.

```suggestion
        clearstatcache(clear_realpath_cache: true, filename: $this->signaturePath);
```